src/instrument/devices/ad_common.py

```diff
@@ -22,36 +22,40 @@
 import time
 from pathlib import PurePath
 
-from apstools.devices import AD_EpicsFileNameHDF5Plugin
-from apstools.devices import AD_plugin_primed
-from apstools.devices import AD_prime_plugin2
-from apstools.devices import CamMixin_V34
-from ophyd import ADComponent
-from ophyd import EpicsSignal
-from ophyd import EpicsSignalRO
-from ophyd import EpicsSignalWithRBV
-from ophyd.areadetector import CamBase
-from ophyd.areadetector import DetectorBase
-from ophyd.areadetector import EigerDetectorCam
-from ophyd.areadetector import SimDetectorCam
-from ophyd.areadetector import SingleTrigger
-from ophyd.areadetector.plugins import CodecPlugin_V34
-from ophyd.areadetector.plugins import FileBase
-from ophyd.areadetector.plugins import ImagePlugin_V34
-from ophyd.areadetector.plugins import OverlayPlugin_V34
-from ophyd.areadetector.plugins import ProcessPlugin_V34
-from ophyd.areadetector.plugins import PluginBase_V34
-from ophyd.areadetector.plugins import PvaPlugin_V34
-from ophyd.areadetector.plugins import ROIPlugin_V34
-from ophyd.areadetector.plugins import StatsPlugin_V34
-from ophyd.areadetector.plugins import TransformPlugin_V34
+from apstools.devices import (
+    AD_EpicsFileNameHDF5Plugin,
+    AD_plugin_primed,
+    AD_prime_plugin2,
+    CamMixin_V34,
+)
+from ophyd import ADComponent, EpicsSignal, EpicsSignalRO, EpicsSignalWithRBV
+from ophyd.areadetector import (
```

Comment:
And these lines are great examples of why to use `isort --sl`.

Reply:
did it